docs/guide/rest-resources.md

```diff
@@ -244,4 +244,41 @@ will also include the pagination information by the following HTTP headers:
 * `X-Pagination-Per-Page`: The number of resources in each page;
 * `Link`: A set of navigational links allowing client to traverse the resources page by page.
 
+Since collection in REST APIs is a data provider, it shares all data provider features i.e. pagination and sorting.
+
 An example may be found in the [Quick Start](rest-quick-start.md#trying-it-out) section.
+
+### Filtering collections
+
+Since 2.0.13 Yii provides a facility to filter collections. In case you are using `ActiveController` you can enjoy it
+right away. An example may be found in the [Quick Start](rest-quick-start.md#trying-it-out) section. In case you're
+implementing an enpoint yourself, filtering could be done like the following:
+
+```php
+$filter = new ActiveDataFilter([
+    'searchModel' => 'app\models\PostSearch'
+]);
+
+$filterCondition = null;
+
+// if you prefer JSON in request body, use Yii::$app->request->getBodyParams() below:
+if ($filter->load(\Yii::$app->request->get())) { 
```

Comment:
With `yii\base\Model` and `yii\web\Request` we can use `post(null, null)` and `load(... , '')`
```php
$filter->load(\Yii::$app->request->post(), '')
```
for `json` OR form `post` data...

Reply:
Of course. Data source could be anything.

Reply:
Do you think it should be mentioned specifically?

Reply:
> // if you prefer JSON in request body, use Yii::$app->request->getBodyParams() below:
This phrase is so concrete. I think we need weaker mention. 
Or more examples...

Reply:
Hm, Interesting realization of `yii\rest\DataFilter::load()`
```php
    /**
     * @inheritdoc
     */
    public function attributes()
    {
        return [
            $this->filterAttributeName
        ];
    }

    /**
     * @inheritdoc
     */
    public function formName()
    {
        return '';
    }
```
@samdark My be we need add link to this in comment?

@see yii\rest\DataFilter::load()
@see yii\rest\DataFilter::attributes()
@see yii\rest\DataFilter::filterAttributeName